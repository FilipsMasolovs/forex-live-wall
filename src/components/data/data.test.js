import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Data from './data';

const data = [{name: "USD/JPY",price: 112.986,change: -0.002093}]

it('renders without crashing', () => {
  const parent = document.createElement('div')
  ReactDOM.render(<Data data={data} />, parent)
})

it('renders correctly', () => {
  const tree = renderer
    .create(<Data data={data} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});