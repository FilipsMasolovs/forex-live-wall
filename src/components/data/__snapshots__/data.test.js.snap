// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="forex-live-wall--currency-data"
>
  <h2>
    USD/JPY
  </h2>
  <h3>
    Price: 
    112.986
  </h3>
  <h3
    className="negative"
  >
    Change: 
    <span>
      -0.002093
    </span>
  </h3>
</div>
`;
